packages/squiggle-lang/src/reducer/lambda.ts

```diff
@@ -219,23 +197,6 @@ export class BuiltinLambda extends BaseLambda {
 
     throw new REOther(showNameMatchDefinitions());
   }
-
-  defaultInputs(): Input[] {
-    const longestSignature = maxBy(this.signatures(), (s) => s.length) || [];
-    return longestSignature.map((sig, i) => {
-      const name = sig.varName ? sig.varName : `Input ${i + 1}`;
-      return frTypeToInput(sig, i, name);
-    });
-  }
-
-  toCalculator(): Calculator {
-    const inputs = this.defaultInputs();
-    return {
-      fn: this,
-      inputs: inputs,
-      autorun: inputs.length !== 0,
-    };
-  }
```

Comment:
Moved to `fr/calculator.ts`.

I had trouble with circular imports, that ended up unrelated to this, but I think it's still a good change — these are too high-level to be defined in `lambda.ts`.